docs/data.fetch.md

```diff
@@ -0,0 +1,82 @@
+# qgrid filtering interface
+
+Filtering is tightly coupled with ordering, paging and virtualization.
```

Comment:
Yes, all data-related operations are tightly coupled. But we need to hardly define a correlation between `data.rows` and all these fetch functions.